src/faviconData.json

```diff
@@ -1 +1 @@
generator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/android-chrome-512x512.png","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/apple-touch-icon-precomposed.png","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/apple-touch-icon.png","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/browserconfig.xml","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/favicon-16x16.png","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/favicon-32x32.png","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/favicon.ico","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/mstile-144x144.png","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/mstile-150x150.png","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/mstile-310x150.png","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/mstile-310x310.png","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/mstile-70x70.png","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/safari-pinned-tab.svg","https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/package_files/site.webmanifest"],"html_code":"<link rel=\"apple-touch-icon\" sizes=\"180x180\" href=\"%PUBLIC_URL%/apple-touch-icon.png\">\n<link rel=\"icon\" type=\"image/png\" sizes=\"32x32\" href=\"%PUBLIC_URL%/favicon-32x32.png\">\n<link rel=\"icon\" type=\"image/png\" sizes=\"16x16\" href=\"%PUBLIC_URL%/favicon-16x16.png\">\n<link rel=\"manifest\" href=\"%PUBLIC_URL%/site.webmanifest\">\n<link rel=\"mask-icon\" href=\"%PUBLIC_URL%/safari-pinned-tab.svg\" color=\"#000000\">\n<link rel=\"shortcut icon\" href=\"%PUBLIC_URL%/favicon.ico\">\n<meta name=\"msapplication-TileColor\" content=\"#00aba9\">\n<meta name=\"msapplication-TileImage\" content=\"%PUBLIC_URL%/mstile-144x144.png\">\n<meta name=\"msapplication-config\" content=\"%PUBLIC_URL%/browserconfig.xml\">\n<meta name=\"theme-color\" content=\"#000000\">","compression":"false","overlapping_markups":["link[rel=\"apple-touch-icon\"]","link[rel=\"shortcut\"]","link[rel=\"shortcut icon\"]","link[rel=\"icon\",sizes=\"16x16\"]","link[rel=\"icon\",sizes=\"32x32\"]","meta[name=\"msapplication-TileColor\"]","meta[name=\"msapplication-TileImage\"]","meta[name=\"msapplication-config\"]","link[rel=\"manifest\"]","meta[name=\"theme-color\"]","link[rel=\"mask-icon\"]"]},"files_location":{"type":"path","path":"%PUBLIC_URL%"},"preview_picture_url":"https://realfavicongenerator.net/files/2405b83237d05ba8a89cc51971768e1f704a3525/favicon_preview.png","version":"0.16"}
```

Comment:
Please check out this file. `src/faviconData.json` is generated with every build. It is essential for `gulp-real-favicon`. It needs to be present while running gulp for the first time. Although I have added it to gitignore, it's not working.